lib/serverless-tools/cli.rb

```diff
@@ -10,7 +10,7 @@ class CLI < Thor
     method_option :functions, :type => :string, :aliases => "-f", :default => "{}"
     def comment
       comment = Comment.new.build(options[:functions])
-      puts "::set-output name=comment::#{comment}"
+      system("echo \"comment=#{comment}\" >> \"$GITHUB_OUTPUT\"")
```

Comment:
Looking a bit gross, we could just open `ENV["GITHUB_OUTPUT"]` and write to it. Or better yet, …

there's no GitHub Actions Toolkit gem?! oh well, whatever, never mind